test/workflow-orchestration.ts

```diff
@@ -14,6 +14,51 @@ import { PATH_TO_CONTAINER_ARTIFACTS } from '../app/backends/workflow-orchestrat
 import path from 'path';
 import { promises as fs } from 'fs';
 
+describe('when a work item callback request does not return the results to construct the next work item(s)', function () {
+  const collection = 'C1233800302-EEDTEST';
+  hookServersStartStop();
+  const reprojectAndZarrQuery = {
+    maxResults: 2,
+    outputCrs: 'EPSG:4326',
+    interpolation: 'near',
+    scaleExtent: '0,2500000.3,1500000,3300000',
+    scaleSize: '1.1,2',
+    format: 'application/x-zarr',
+    concatenate: false, // Aggregated workflows are tested above
+  };
+
+  hookRangesetRequest('1.0.0', collection, 'all', { query: reprojectAndZarrQuery });
+  hookRedirect('joe');
+
+  it('generates a workflow with 3 steps', async function () {
+    const job = JSON.parse(this.res.text);
+    const workflowSteps = await getWorkflowStepsByJobId(db, job.jobID);
+
+    expect(workflowSteps.length).to.equal(3);
+  });
+
+  describe('when executing a query-cmr work item and no catalog is returned', function () {
+    it('finds the queued work item, but query-cmr fails to return a catalog for the next work items', async function () {
+      const res = await getWorkForService(this.backend, 'harmonyservices/query-cmr:latest');
+      expect(res.status).to.equal(200);
+      const workItem = JSON.parse(res.text);
+      expect(workItem.serviceID).to.equal('harmonyservices/query-cmr:latest');
+      workItem.status = WorkItemStatus.SUCCESSFUL;
+      workItem.results = [];
+      await updateWorkItem(this.backend, workItem);
```

Comment:
It seems weird to me that you are doing this inside an `it` with not following `expect`.

Reply:
Yeah these orchestration tests are like this throughout - I'm not sure how to get access to the work item outside of the `it` block.